### ISSUE TITLE
OCKCareCardViewController - UIAlertController removes progress fill

### ISSUE THREAD

**Issue by bonebox** [carekit-apple/CareKit#77]
Displaying a UIAlertController on top of a OCKCareCardViewController (via delegate method) clears out the progress fill on both the maskImage and smallMaskImage images.

**Comment by harsh62** [comment]
Whenever a modal view is presented the tint color of the views behind it are changed to a gray "dimmed" color to indicate that they are not interactive. 

Does this even needs to be fixed?

**Comment by bonebox** [comment]
I'm not sure what you're asking.  That's entirely different than the issue that I described.

**Comment by harsh62** [comment]
You are talking about this issue right?

<img width="432" alt="screen shot 2016-07-18 at 5 17 05 pm" src="https://cloud.githubusercontent.com/assets/6162866/16930739/8dd440ae-4d0b-11e6-8297-8af45dfcc019.png">

**Comment by bonebox** [comment]
No. I would assume that should be the expected behavior.  This is the issue:

![simulator screen shot jul 18 2016 4 34 57 pm](https://cloud.githubusercontent.com/assets/1120504/16931326/0a9925c4-4d06-11e6-9971-0345d22fcabc.png)

![simulator screen shot jul 18 2016 4 35 19 pm](https://cloud.githubusercontent.com/assets/1120504/16931325/0a99185e-4d06-11e6-8dae-3a71d36e12bd.png)

**Comment by harsh62** [comment]
ON which delegate method does it happen?

**Comment by bonebox** [comment]
On both `careCardViewController(_:, didSelectRowWithInterventionActivity:)` and `careCardViewController(_:, didSelectButtonWithInterventionEvent:)`.
